drivers/CoreVideo/include/interface/Font.hpp

```diff
@@ -4,27 +4,25 @@
 
 #pragma once
 
-// #include <cstdint>
+#include <cstdint>
 
-#include "CGColor.hpp"
-#include "CGPixel.hpp"
+namespace leka {
+
+struct CGColor;
+struct Character;
```

Comment:
Nice! 👍 

In the future, I still have the wish to put all those "user types" in the main `include` directory and have our code depend on it directly.

This is something discussed by Tony in the Cpp.chat episode https://www.youtube.com/watch?v=WRMhO9Gn5GM

Very interesting talk btw.